code/obj/item/fitness.dm

```diff
@@ -267,18 +267,16 @@
 				H.sims.affectMotive("fun", 1)
 		spam_flag = 1
 		if (narrator_mode)
-			playsound(user, 'sound/vox/duct.ogg', 50, 1)
+			playsound(user, 'sound/vox/screamsound.ogg', 50, 1)
```

Comment:
try `sound/vox/tunnel.ogg`

Reply:
I think it's funnier if the vox screams too

Reply:
you're fundamentally misunderstanding what narrator mode is, i think

Reply:
Isnt narrator mode a admin toggle that then has the VOX say actions and shit?

Reply:
yes. as in, it says the name of the thing. 
it shouldnt be screaming.

Reply:
set this to a word, or remove it, and it'll merge.